src/state/AppState.ts

```diff
@@ -32,6 +33,22 @@ export interface AnalysisGame {
   currentMove: number;
 }
 
+export interface HowToPlay {
+  open: boolean;
+  stepNumber: number;
+  maxStepNumber: number;
+  stateNumber: number;
+  text: React.ReactNode;
```

Comment:
Storing a ReactNode in Redux is generally not a good practise and discouraged: https://github.com/reduxjs/redux/issues/1248
What alternatives do we have when we want to display a custom component next to the displayed text that is saved in the store?
```
    text: (
      <>
        Each turn the previous move of your opponent dictates in which board (
        <HowToPlayIndicator />) you can move next
      </>
    ),
```

Reply:
you could create a function resolving the text from the stepNumber

Reply:
Makes sense!
So something like this:
```
export const getCurrentHowToPlayText = createSelector(
  [getHowToPlay],
  (howToPlay: HowToPlay) => {
    return howToPlaySteps[howToPlay.stepNumber].text;
  },
);
```